setup.py

```diff
@@ -43,7 +43,13 @@
     sources=[
         'src/uharfbuzz/_harfbuzz.pyx',
         'harfbuzz/src/harfbuzz.cc',
+        'harfbuzz/src/hb-subset-cff-common.cc',
+        'harfbuzz/src/hb-subset-cff1.cc',
+        'harfbuzz/src/hb-subset-cff2.cc',
+        'harfbuzz/src/hb-subset-input.cc',
+        'harfbuzz/src/hb-subset-plan.cc',
         'harfbuzz/src/hb-subset-repacker.cc',
+        'harfbuzz/src/hb-subset.cc',
         'harfbuzz/src/graph/gsubgpos-context.cc',
```

Comment:
You can do it the otherway, use `harfbuzz-subset.cc` + hb-coretext.cc, these platform integration files are unlikely to change.